include/libtorrent/aux_/resolve_links.hpp

```diff
@@ -32,29 +32,29 @@ namespace libtorrent::aux {
 	{
 		struct TORRENT_EXTRA_EXPORT link_t
 		{
-			std::shared_ptr<const torrent_info> ti;
+			std::shared_ptr<torrent_info const> ti;
```

Comment:
Is it stylistic best practice to prefer a const at the end now, instead of at the beginning?

Reply:
I subscribe to "east const": https://hackingcpp.com/cpp/design/east_vs_west_const.html